src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -127,6 +133,78 @@ def __getattribute__(self, name):
         return val
 
 
+def in_cloud_console():
+    return os.environ.get('ACC_CLOUD', None)
+
+
+def _ssl_context():
+    if sys.version_info < (3, 4) or (in_cloud_console() and platform.system() == 'Windows'):
+        try:
+            return ssl.SSLContext(ssl.PROTOCOL_TLS)  # added in python 2.7.13 and 3.6
+        except AttributeError:
+            return ssl.SSLContext(ssl.PROTOCOL_TLSv1)
+
+    return ssl.create_default_context()
+
+
+def urlretrieve(url):
+    req = urlopen(url, context=_ssl_context())
+    return req.read()
+
+
+def get_ossrdbms_resource_id(cloud_name):
+    ossrdbms_mapper = {
+        'AzureCloud': 'https://ossrdbms-aad.database.windows.net',
```

Comment:
What is ossrdbms?

Reply:
OSS relational DB offerings in Azure. A blog at https://azure.microsoft.com/en-us/blog/happy-birthday-to-managed-open-source-rdbms-services-in-azure/ has some context.

OSS RDBMS config currently exists in CLI, so there's nothing new here.